concrete-core/src/specification/engines/glwe_ciphertext_creation.rs

```diff
@@ -0,0 +1,62 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::GlweCiphertextEntity;
+use concrete_commons::parameters::PolynomialSize;
+
+engine_error! {
+    GlweCiphertextCreationError for GlweCiphertextCreationEngine @
+    EmptySlice => "The container used to create the GLWE ciphertext is of length 0!",
+    InvalidSliceSize => "The length of the container used to create the GLWE ciphertext \
```

Comment:
Here, I would replace `Slice` by `Container`

Reply:
Yes you are right!